packages/typescript-estree/src/create-program/createProjectService.ts

```diff
@@ -58,6 +60,40 @@ export function createProjectService(
     jsDocParsingMode,
   });
 
+  if (typeof options === 'object' && options.defaultProject) {
+    try {
+      const configRead = tsserver.readConfigFile(
```

Comment:
Fun fact: `ts.readConfigFile` has two unhappy paths:
* If the config file is valid JSON but not right for TypeScript, it returns an object
* If the config file is invalid JSON, it throws

🙌 error handling!